cumulus/client/consensus/aura/src/collators/lookahead.rs

```diff
@@ -386,11 +386,7 @@ where
 						None,
 						(parachain_inherent_data, other_inherent_data),
 						params.authoring_duration,
-						// Set the block limit to 50% of the maximum PoV size.
-						//
-						// TODO: If we got benchmarking that includes the proof size,
-						// we should be able to use the maximum pov size.
-						(validation_data.max_pov_size / 2) as usize,
+						validation_data.max_pov_size as usize,
```

Comment:
(The PR description already says so, I just want to double check if there aren't any dragons hidden somewhere)

Reply:
For example, the PoV also includes some relay chain storage items required during execution, not sure if this is accounted for here.

Reply:
IIUC the need to benchmark proof sizes was superseded by the proof recording, which accounts for the exact proof sizes, but I'm asking to double-check me here because I'm also not 200% sure I understand everything :sweat_smile: 

Multiple Zombienet tests ran without a problem, but a burn-in on Versi might also be a good idea.

Reply:
> For example, the PoV also includes some relay chain storage items required during execution, not sure if this is accounted for here.

I checked and looks to be part of the inherent data of the parachain, so we should be good.